resources/apiserver-proxy/templates/service.yaml

```diff
@@ -1,7 +1,34 @@
+---
+# Secure port service
 apiVersion: v1
 kind: Service
 metadata:
+  name: {{ template "name" . }}-ssl
+  namespace: {{ .Release.Namespace }}
+{{- if eq .Values.global.certificates.type "gardener" }}
   annotations:
+    cert.gardener.cloud/secretname: {{ template "name" . }}-tls-cert
+    dns.gardener.cloud/class: garden
+    dns.gardener.cloud/dnsnames: {{ printf "apiserver.%s" .Values.global.ingress.domainName }}
+{{- end }}
+spec:
+  {{- if .Values.global.isLocalEnv }}
```

Comment:
Do not use isLocalEnv for network settings. See: https://github.com/kyma-project/kyma/issues/9442

Reply:
Fixed in d2f0cfa